source/iostreams.tex

```diff
@@ -6419,6 +6420,17 @@
 \tcode{*this}.
 \end{itemdescr}
 
+\indexlibrarymember{operator<<}{basic_ostream}%
+\begin{itemdecl}
+basic_ostream& operator<<(const volatile void* p);
```

Comment:
Shouldn't we use "val" instead of "p" and combine this with the previous itemdecl, similar to how the non-volitile "val" version was?

Reply:
It's a bit awkward that the current `const void*` overload uses `p` in the header synopsis but `val` in the description, but regardless, using `p` in both places for the new overload seems good.